fem/ceed/util.cpp

```diff
@@ -374,6 +551,15 @@ int CeedOperatorGetActiveField(CeedOperator oper, CeedOperatorField *field)
    return 0;
 }
 
+int CeedOperatorGetActiveBasis(CeedOperator oper, CeedBasis *basis)
```

Comment:
This method already exists in libCEED: https://libceed.readthedocs.io/en/latest/api/internal/CeedOperator/?highlight=Active#c.CeedOperatorGetActiveBasis